dask/dataframe/utils.py

```diff
@@ -731,7 +733,7 @@ def valid_divisions(divisions):
     if not isinstance(divisions, (tuple, list)):
         return False
 
-    if pd.isnull(divisions).any():
+    if pd.Series(divisions).isnull().any():
```

Comment:
It'd be good to link reference the corresponding issue in `pandas`. Also, minor comment, I think just casting to a `list` will be a bit easier for future readers to reason about. 

```suggestion
    # Cast tuples to lists as `pd.isnull` treats them differently
    # https://github.com/pandas-dev/pandas/issues/52283
    if isinstance(divisions, tuple):
        divisions = list(divisions)
    if pd.isnull(divisions).any():
```